src/comparator.html

```diff
@@ -0,0 +1,252 @@
+<!DOCTYPE html>
+<html lang="en">
+  <title>Elastic Maps Service Style Compare</title>
+  <style>
+    h2 {
+      z-index: 9999;
+    }
+    .drag {
+      opacity: 0.5;
+      transition: opacity 0.15s ease-in;
+    }
+    .mapboxgl-ctrl-attrib {
+      display: none;
+    }
+  </style>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="https://unpkg.com/tachyons/css/tachyons.min.css" />
+  <link rel="stylesheet" href="https://api.mapbox.com/mapbox-gl-js/v1.11.0/mapbox-gl.css" />
+  <link
+    rel="stylesheet"
+    type="text/css"
+    href="https://unpkg.com/dialog-polyfill@0.5.1/dist/dialog-polyfill.css"
+  />
+  <body>
+    <dialog class="w-50">
+      <form method="dialog">
+        <input type="submit" value="x" class="fr ba br1" />
+      </form>
+      <article class="br2 mv4 w-100 center">
```

Comment:
```suggestion
      <article class="br2 mv1 w-100 center">
```

I'd put a smaller margin here